Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -764,6 +763,10 @@ void FontCache::platformInit()
 {
     CFNotificationCenterAddObserver(CFNotificationCenterGetLocalCenter(), this, &fontCacheRegisteredFontsChangedNotificationCallback, kCTFontManagerRegisteredFontsChangedNotification, nullptr, CFNotificationSuspensionBehaviorDeliverImmediately);
 
+    CFNotificationCenterAddObserver(CFNotificationCenterGetLocalCenter(), this, &fontCacheRegisteredFontsChangedNotificationCallback, kAXSEnhanceTextLegibilityChangedNotification, nullptr, CFNotificationSuspensionBehaviorDeliverImmediately);
+
+    CFNotificationCenterAddObserver(CFNotificationCenterGetLocalCenter(), this, &fontCacheRegisteredFontsChangedNotificationCallback, getUIContentSizeCategoryDidChangeNotificationName(), 0, CFNotificationSuspensionBehaviorDeliverImmediately);
+
```

Comment:
Does this belong in this patch? Seems unrelated to the code moving.

Reply:
Whoopsie!

Reply:
The bottom one is necessary (and is a move) - see the previous implementation of `RenderThemeIOS::RenderThemeIOS()`.

The top one is a mistake.